packages/stylify/src/Compiler/Compiler.ts

```diff
@@ -199,19 +199,40 @@ export class Compiler {
 		selector: string,
 		componentConfig: ComponentSelectorsType|ComponentConfigInterface
 	): Compiler {
-		if (selector in this.components) {
+		if (selector.includes(',')) {
+			selector.split(',').forEach((selector) => {
+				this.addComponent(selector.trim(), componentConfig);
+			});
+			return;
+		}
+
+		const selectorIsComponent = selector in this.components;
+
+		if (selectorIsComponent && this.components[selector].processed === true) {
+			const info = `You are trying to overwrite component ${selector} that has already been processed.`;
```

Comment:
I would rather used `You are trying to configure...` instead of `overwrite`.
And add quotes around the "${selector}".